packages/devtools-components/src/tests/tree.js

```diff
@@ -653,14 +778,27 @@ function getTreeNodes(wrapper) {
   return wrapper.find(".tree-node");
 }
 
-function simulateKeyDown(wrapper, key) {
+function simulateKeyDown(wrapper, key, options) {
   wrapper.simulate("keydown", {
     key,
     preventDefault: () => {},
-    stopPropagation: () => {}
+    stopPropagation: () => {},
+    ...options
   });
 }
 
+function renderItemWithFocusableContent(x, depth, focused, arrow) {
+  let children = [{}, arrow, focused ? "[" : null, x, focused ? "]" : null];
+  if (x === "L") {
+    children = [
+      ...children,
+      dom.a({ id: "active-anchor", href: "#" }, "anchor")
+    ];
+  }
+
+  return dom.div(...children);
```

Comment:
If we remove the `{}` from the `children` array, we can change this line into: 

```suggestion
  return dom.div({}, children);
```